example/hands-on/config/linkstatus_config.yml

```diff
@@ -0,0 +1,3 @@
+:linkstatus_parallel_num: 8
```

Comment:
why do you commit this file?

Reply:
I commit all files which are created in this example.